public/tutorials/setup-for-erc20-upgrade.sh

```
@@ -0,0 +1,77 @@
+if [ -z $1 ]
+then
+  echo Supersim
+  # This is a well-known development private key – never use in production
+  PRIVATE_KEY=0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80
+  URL_CHAIN_A=http://localhost:9545
+  URL_CHAIN_B=http://localhost:9546
+else
+  echo Devnet
+  PRIVATE_KEY=$1
+  URL_CHAIN_A=https://interop-alpha-0.optimism.io
+  URL_CHAIN_B=https://interop-alpha-1.optimism.io
```

Shellcheck (0.10.0)

[warning] URL_CHAIN_B appears unused. Verify use (or export if used externally).

(SC2034)